content/id/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -0,0 +1,177 @@
+
+Untuk membuat klaster yang kompatibel dengan HA, kamu harus mengatur tanda ini pada skrip `kube-up`: 
+
+* `MULTIZONE=true` - untuk mencegah penghapusan replika master kubelet dari zona yang berbeda dengan zona bawaan server.
+Ini diperlukan jika kamu ingin menjalankan replika master pada zona berbeda, dimana hal ini disarankan.
+
+* `ENABLE_ETCD_QUORUM_READ=true` - untuk memastikan bahwa pembacaan dari semua server API akan mengembalikan sebagian besar data terbaru.
+Jika benar, bacaan akan diarahkan ke replika pemimpin dari etcd.
+Menetapkan nilai ini menjadi `true` bersifat opsional: pembacaan akan lebih dapat diandalkan tetapi juga akan menjadi lebih lambat.
+
+Secara opsional, kamu dapat menentukan zona GCE tempat dimana replika master pertama akan dibuat.
+Atur tanda berikut:
+
+* `KUBE_GCE_ZONE=zone` - zona tempat dimana replika master pertama akan berjalan.
+
+Contoh perintah berikut ini mengatur klaster yang kompatibel dengan HA pada zona GCE europe-west1-b:
+
+```shell
+MULTIZONE=true KUBE_GCE_ZONE=europe-west1-b  ENABLE_ETCD_QUORUM_READS=true ./cluster/kube-up.sh
+```
+
+Perhatikan bahwa perintah di atas digunakan untuk membuat klaster dengan sebuah master;
+Namun, kamu bisa menambahkan replika master baru ke klaster dengan perintah berikutnya.
+
+
+## Menambahkan replika master yang baru
+
+Setelah kamu membuat klaster yang kompatibel dengan HA, kamu bisa menambahkan replika master ke sana.
+Kamu bisa menambahkan replika master dengan menggunakan skrip `kube-up` dengan tanda berikut ini:
+
+* `KUBE_REPLICATE_EXISTING_MASTER=true` - untuk membuat replika dari master yang sudah ada.
+
+* `KUBE_GCE_ZONE=zone` - zona dimana replika master itu berjalan.
+Region ini harus sama dengan region dari zona replika yang lain.
+
+Kamu tidak perlu mengatur tanda `MULTIZONE` or `ENABLE_ETCD_QUORUM_READS`,
+karena tanda itu diturunkan pada saat kamu memulai klaster yang kompatible dengan HA.
+
+Contoh perintah berikut ini mereplikasi master pada klaster sebelumnya yang kompatibel dengan HA:
+
+```shell
+KUBE_GCE_ZONE=europe-west1-c KUBE_REPLICATE_EXISTING_MASTER=true ./cluster/kube-up.sh
+```
+
+## Menghapus replika master
+
+Kamu dapat menghapus replika master dari klaster HA dengan menggunakan skrip `kube-down` dengan tanda berikut:
+
+* `KUBE_DELETE_NODES=false` - untuk mencegah penghapusan kubelet.
+
+* `KUBE_GCE_ZONE=zone` - zona dimana replika master akan dihapus.
+
+* `KUBE_REPLICA_NAME=replica_name` - (opsional) nama replika master yang dihapus.
+Jika kosong: replika mana saja dari zona yang diberikan akan dihapus.
+
+Contoh perintah berikut menghapus replika master dari klaster HA yang sudah ada sebelumnya:
+
+```shell
+KUBE_DELETE_NODES=false KUBE_GCE_ZONE=europe-west1-c ./cluster/kube-down.sh
+```
+
+## Mengatasi replika master yang gagal
+
+Jika salah satu replika master di klaster HA kamu gagal,
+praktek terbaik adalah menghapus replika dari klaster kamu dan menambahkan replika baru pada zona yang sama.
```

Comment:
```suggestion
praktik terbaik adalah menghapus replika dari klaster kamu dan menambahkan replika baru pada zona yang sama.
```

Reply:
"praktek" is more official 
https://kbbi.kemdikbud.go.id/entri/praktek

Reply:
> "praktek" is more official
> https://kbbi.kemdikbud.go.id/entri/praktek

If you open https://kbbi.kemdikbud.go.id/entri/praktik then it is listed "praktek" as tidak baku